### ISSUE TITLE
Error when transforming httpclient spans

### ISSUE THREAD

**Issue by FrankChen021** [FrankChen021/bithon#607]
![image](https://github.com/FrankChen021/bithon/assets/6525742/65e4fe3b-5fe8-4be0-b6fe-84a8a0c23ef1)

At TraceSpanTransformer.java
```java
        if (StringUtils.hasText(uri)
            && SpanKind.CLIENT.name().equals(span.getKind())
            && "httpclient".equals(span.getName())) {
            try {
                URL url = new URL(uri);
                String path = url.getPath();
                String query = url.getQuery();

                tags.remove(Tags.Http.URL);
                tags.put(Tags.Http.TARGET, query == null ? path : (path + "?" + query));
                tags.put(Tags.Net.PEER, url.getHost() + ":" + url.getPort());
```

**Comment by FrankChen021** [comment]
TraceTopoBuilder should be also adjusted to use correct Tag name

```java
            if (childSpan.children.size() == 0) {
                String uriText = null;
                if (SpanKind.CLIENT.name().equals(childSpan.getKind())) {
                    uriText = childSpan.getTag(Tags.Http.URL);
                } else if (SpanKind.PRODUCER.name().equals(childSpan.getKind())) {
                    uriText = childSpan.getTag("uri");
                }

                if (uriText != null) {
                    try {
                        URI uri = new URI(uriText);
                        TraceSpan next = new TraceSpan();
                        next.setAppName(uri.getScheme());
                        next.setInstanceName(uri.getHost() + ":" + uri.getPort());
                        this.addLink(childSpan, next).incrCount();
                    } catch (URISyntaxException ignored) {
                    }
                }
            }
```

**Comment by FrankChen021** [comment]
Fixed by #608
